src/symmeplot/matplotlib/scene.py

```diff
@@ -144,6 +145,51 @@ def plot(self, prettify=True, ax_scale=1.5):
             self.auto_zoom()
             self.axes.set_aspect("equal", adjustable="box")
 
+    def set_plot_as_2d(self, frame: ReferenceFrame | None = None) -> None:
+        """Change the axis to an orhogonal projection making the view seemingly 2D.
+
+        Parameters
+        ----------
+        frame : ReferenceFrame, optional
+            Reference frame w.r.t. which the axis view is oriented aligning the users
+            view with the XY plane. The default is the inertial frame of the scene.
```

Comment:
As detailed on [this page](https://matplotlib.org/stable/api/toolkits/mplot3d/view_angles.html) the `azimuth`, `elevation`, and `roll` angles are defined as a `zyx` body-fixed rotation, where the camera by default looks at the YZ plane. We could either choose to use the same default or we'll have to modify the angles such that we'll be looking at the XY plane as we state here.

Reply:
Although personally I like to plot in the xy plane, it somehow makes sense in my head, it is better to stick to convention. So updated the function to reflect this.